src/ADAL.PCL.iOS/TokenCachePlugin.cs

```diff
@@ -26,12 +29,66 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory
 {
     internal class TokenCachePlugin : ITokenCachePlugin
     {
+        private const string LocalSettingsContainerName = "ActiveDirectoryAuthenticationLibrary";
```

Comment:
I am not sure if we ever need to switch from Xamarin to Native while expecting all the previous tokens be still usable! However, if you still insist, then decode the name you suggest for me:

1. Why MSOpenTech? We do not publish our binaries under that name?
2. What is .1 at the end? Is this the version? If so, .NET token cache schema is already 2.